gcp_variant_transforms/libs/bigquery_util.py

```diff
@@ -40,17 +40,16 @@
 _TOTAL_BASE_PAIRS_SIG_DIGITS = 4
 _PARTITION_SIZE_SIG_DIGITS = 1
 
+START_POSITION_COLUMN = 'start_position'
 _BQ_CREATE_PARTITIONED_TABLE_COMMAND = (
-    'bq mk --table '
-    '--range_partitioning=start_position,0,{TOTAL_BASE_PAIRS},{PARTITION_SIZE} '
+    'bq mk --table --range_partitioning='
+    '{PARTITION_COLUMN},0,{TOTAL_BASE_PAIRS},{PARTITION_SIZE} '
```

Comment:
If the partition column can vary, does it still make sense to call end of range partition TOTAL_BASE_PAIRS?